project_categ/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Project Category
+================
+
+* This module create a parent Category (Tag). E.g. "System Type"..
+* Create categories to be made available as child. E.g. "Computer", "Printer", ....
+* Finally, On the Project form, Other Info tab, set the "Root Category".
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/10.0
+
+Known issues / Roadmap
+======================
+
+* 
+
+Bug Tracker
```

Comment:
Please update following the standard README template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst